App/Redux/UIRedux.js

```diff
@@ -0,0 +1,40 @@
+import { createReducer, createActions } from 'reduxsauce'
```

Comment:
Created this redux config for tracking user interaction flows, like sharing an image. It isn't persistent.